source/configuration_manual/mail_location/obox/http_based_object_storages.rst

```diff
@@ -35,7 +35,14 @@ The parameters common to all object storages include:
 |                                 |                                                                                                                               |              |
 |                                 |.. deprecated:: 2.3.0                                                                                                          |              |
 +---------------------------------+-------------------------------------------------------------------------------------------------------------------------------+--------------+
-| addhdr=<name>:<value>           |Add the specified header to all HTTP requests. This can only be specified once. This may be useful for load balancing          | none         |
+| addhdr=<name>:<value>           |Add the specified header to all HTTP requests. This may be useful for load balancing.                                          | none         |
+|                                 |                                                                                                                               |              |
+|                                 |.. versionadded:: 2.3.10 Can be specified multiple times.                                                                      |              |
++---------------------------------+-------------------------------------------------------------------------------------------------------------------------------+--------------+
+| addhdrvar=<name>:<variable>     |Add the specified header to all HTTP requests and set the value to the expanded variables value.                               | none         |
+|                                 |This may be useful for load balancing.                                                                                         |              |
```

Comment:
Remove the "This may be useful for load balancing" .. I think also from addhdr. That's only one very specific use case for CDMI which isn't really even used anymore.